### PR TITLE
add intolerance-filter information

### LINKED ISSUE #24: User Story 13: Intolerance Information
## Value statement - a requirement, not a solution
As a **user** 
I need **more information about the different intolerances** 
so that **I know what they mean exactly** 

## Description (Text, Scribble, Wireframe, Design)
image

## Acceptance criteria
- [ ] you can see an info-button near the navBar
- [ ] clicking on the button opens a modal with more information about the considered intolerances 

## Tasks
- [ ] add an info-button
- [ ] create a information-modal that opens when clicking on the button

## Size
medium

### PR DESCRIPTION
**Pull request by s-kond**
fixes #24

### CODE REVIEW

File: src/components/IntoleranceFilterTable.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
+import styled from "styled-components";
+
+export default function IntoleranceFilterInformation() {
+  return (
+    <StyledTable>
```
Comment (Kheirah) [inline src/components/IntoleranceFilterTable.js:5]: I'm wondering whether there was a more automated way of generating the table content, like fetching for all possible intolerances?

**Review by Kheirah — APPROVED** [review]
LGTM!

File: src/components/IntoleranceFilterTable.js (hunk 1 above)
Comment (s-kond) [reply src/components/IntoleranceFilterTable.js:5]: Would've been great, but unfortunately the API doesn't provide the definitions for the intolerances - I had to copy & paste them from the documentation...

Comment (Kheirah) [reply src/components/IntoleranceFilterTable.js:5]: I see, all right then :)

**Pull request merged**